### PR TITLE
Pipeline work 

### PR DESCRIPTION
**Pull request by DraganaMana**
We changed some small things and commented some others in order to make it run on NeuroSpin data.

### CODE REVIEW

File: 01-import_and_filter.py [hunk 1]
```diff
@@ -31,11 +32,11 @@ def run_filter(subject):
     raws = []
     for run in config.runs:
 
-        # read bad channels for run from config
-        if run:
-            bads = config.bads[subject][run]
-        else:
-            bads = config.bads[subject]
+         # read bad channels for run from config
```
Comment (SophieHerbst) [inline 01-import_and_filter.py:33]: @DraganaMana you should not have to comment that out. Does it break if you put no runs/ bad channels?

Comment (DraganaMana) [reply 01-import_and_filter.py:33]: Yes, we tried a few options: omitting bads, leaving it bads = dict() , bads = {}, but we still get an error such as: 
  File "C:/Users/Dragana/Repositories/CogBrainDyn_MEG_Pipeline/01-import_and_filter.py", line 39, in run_filter
    bads = config.bads[subject]

Comment (DraganaMana) [reply 01-import_and_filter.py:33]: I tried now by defining the MISC 001, MISC 002 as bad channels, and it's still not working.

Comment (SophieHerbst) [reply 01-import_and_filter.py:33]: It needs t be a dict with the subject name and the bad channels:
bads = dict(my_subject=['MEG 2443', 'EEG 053'])
Please try that and put it back, you will need it. 
For the new data, you can put bad channels per run.

File: 02-apply_maxwell_filter.py [hunk 2]
```diff
@@ -20,7 +20,6 @@
 
 import mne
 from mne.parallel import parallel_func
-from mne.preprocessing import maxwell_filter
```
Comment (SophieHerbst) [inline 02-apply_maxwell_filter.py]: I think we need maxfilter, no?

File: 01-import_and_filter.py (hunk 1 above)
Comment (DraganaMana) [reply 01-import_and_filter.py:33]: @Izem0

File: config.py [hunk 3]
```diff
@@ -103,11 +104,11 @@
...
-l_freq = 1
+l_freq = 0.01
 
 # ``h_freq``  : the high-frequency cut-off in the lowpass filtering step.
 # Keep it None if no lowpass filtering should be applied.
-h_freq = 45
+h_freq = 100
 
 
 ###############################################################################
```
Comment (DraganaMana) [inline config.py:116]: @SophieHerbst I get an error when I try to run the 02-apply_maxwell_filter.py: 
  File "C:\Users\Dragana\Anaconda3\lib\site-packages\mne\preprocessing\maxwell.py", line 436, in maxwell_filter
    if read_lims[-1] != end:

IndexError: index -1 is out of bounds for axis 0 with size 0

Comment (SophieHerbst) [reply config.py:116]: Hi Dragana,
I vaguely remember having had that problem with an MEG file for which I had switched the recording on and off, causing discontinuities. I don't remember what I did though..
Can you try to visualize the data using the first script and look whether there are any boundaries (marked by red lines)? If they appear largely before your first triggers, I would crop the file manually and save it again. 
I am not an NS tomorrow, but available in the morning. 
Sophie

Comment (DraganaMana) [reply config.py:116]: Hello Sophie, We clicked twice on 2 occasions on Record raw and there are 2 files per run. But one of them is much less memory heavy. I proceeded to analyze the 'good' runs, and will cut the one which has the boundaries.

**Comment by virvw** [comment]
an MEG recording will not go over 2GB if it does, the file is automatically
recorded in two, three, etc parts. when you read the first part, mne
automatically linked to the other

Virginie van Wassenhove

**Comment by DraganaMana** [comment]
@virvw Thanks! This is exactly the case as the first files have 2GB

**Comment by DraganaMana** [comment]
Last error when running script 04:

  File "C:\Users\Dragana\Anaconda3\lib\site-packages\mne\evoked.py", line 629, in _check_decim
    if decim < 1 or decim != int(decim):

TypeError: '<' not supported between instances of 'NoneType' and 'int'

**Comment by DraganaMana** [comment]
Error when running 06a-apply_ica.py:

  File "C:/Users/Dragana/Repositories/CogBrainDyn_MEG_Pipeline/06a-apply_ica.py", line 163, in run_evoked
    list(config.rejcomps_man[subject][ch_type]))

AttributeError: module 'config' has no attribute 'rejcomps_man'

**Comment by SophieHerbst** [comment]
rejcomps_man asks for ICA components you want to reject. Does your script 05 create the html with the components plotted?

**Pull request closed without merging**